docs/API/InstrumentFormat.md

```diff
@@ -138,7 +181,7 @@ as follows. It denotes a group of values of which the user must select one optio
             ...
         },
         "AllowMultiple"   : boolean,
-        "RequireResponse" : boolean
+        "RequireResponse" : string, /* LORIS Logic Parser formula */
```

Comment:
The phrase "RequireResponse" seems inherently boolean-y to me. If it were to be a string, I'd intuitively interpret it as "Error message to display", not a formula.

I'd think a separate "RequireFormula" or something of the sort in the (hypothetical, new) DisplayRules section would make more sense and leave RequireResponse alone.